doc/api/synopsis.md

```diff
@@ -9,9 +9,43 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Firstly, make sure you have downloaded Node.js from [Node.js Official website](http://nodejs.org/#download).
+Then, follow this [installation guide](https://nodejs.org/en/download/package-manager/).
+
+
+Now, create an empty project folder called `projects`, navigate into it:
```

Comment:
Can you make it more clear that your folder doesn't _have_ to be called `projects`? Maybe:

>Create a directory to put your code in, for example to create a `projects` folder in your home directory you can run these commands in a terminal: